src/Control/Carrier/Error/Either.hs

```diff
@@ -43,7 +43,7 @@ runError (ErrorC m) = runExceptT m
 
 -- | @since 0.1.0.0
 newtype ErrorC e m a = ErrorC (ExceptT e m a)
-  deriving (Applicative, Functor, Monad, Fail.MonadFail, MonadFix, MonadIO, MonadTrans)
+  deriving (Algebra (Error e :+: sig), Applicative, Functor, Monad, Fail.MonadFail, MonadFix, MonadIO, MonadTrans)
```

Comment:
This is one of the only cases where we define a carrier wrapping another one just so that we can customize some instance _other_ than `Algebra`, and so is likewise one of a small number of cases where we can derive an `Algebra` instance.